extensions/amp-story/1.0/amp-story-cta-layer.js

```diff
@@ -60,4 +62,19 @@ export class AmpStoryCtaLayer extends AmpStoryBaseLayer {
       ctaLinks[i].setAttribute('target', '_blank');
     }
   }
+
+  /**
+   * CTA links or buttons are not allowed on the first amp-story page. Remove
+   * the amp-story-cta-layer if it is found on the first page of the story.
+   * @private
+   */
+  checkAndRemoveLayerIfOnFirstPage_() {
+    if (matches(this.element,
+        'amp-story-page:first-of-type amp-story-cta-layer')) {
```

Comment:
nit: slight performance optimization: `amp-story-page:first-of-type > amp-story-cta-layer`